test/test_oneview_server_profile.py

```diff
@@ -1874,6 +1877,36 @@ def test_should_not_update_when_mac_not_passed_in_deployment_settings(self, mock
             ansible_facts=mock_facts
         )
 
+    # @mock.patch.object(ResourceComparator, 'compare')
```

Comment:
This should be commented out?

Reply:
Definitely.... not. It should be removed. Nice catch.